Modules/posixmodule.c

```diff
@@ -5228,6 +5310,59 @@ os_spawnve_impl(PyObject *module, int mode, path_t *path, PyObject *argv,
 #endif /* HAVE_SPAWNV */
 
 
+#ifdef HAVE_FORK
+/*[clinic input]
+os.register_at_fork
```

Comment:
Maybe registering fork handlers require doing other preparation work. It is easy to test if `os.register_at_fork` exists, and this is the only way for testing if registering the fork handlers is needed or can be just skipped (with additional preparation work).